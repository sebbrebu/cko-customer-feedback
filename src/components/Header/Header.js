import React from 'react';
import './Header.css';

const header = (props) => {
    return (
        <header className='Header'>
            <h1>{props.title}</h1>
        </header>
    );
}

export default header;