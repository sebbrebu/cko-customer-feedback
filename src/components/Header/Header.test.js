import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Header from './Header';

afterEach(cleanup);

/*Seb: for this component, we'll just test against a snapshot 
       as the component it's not likely to change, and if it does, we'll be warned by this test.
*/
it('renders', () => {
    const { asFragment } = render(<Header title="Customer Feedback" />);
    expect(asFragment()).toMatchSnapshot();
});