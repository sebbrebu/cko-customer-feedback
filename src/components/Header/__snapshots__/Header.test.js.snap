// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <header
    class="Header"
  >
    <h1>
      Customer Feedback
    </h1>
  </header>
</DocumentFragment>
`;
